modules/guardrails-orchestrator-configuring-regex.adoc

```diff
@@ -0,0 +1,122 @@
+:_module-type: PROCEDURE
+
+ifdef::context[:parent-context: {context}]
+[id="guardrails-orchestrator-configuring-regex_{context}"]
+= Configuring the regex detector
+[role='_abstract']
+
+The following example demonstrates how to enable the regex detector with the `GuardrailsOrchestrator` service.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have downloaded and installed the OpenShift command-line interface (CLI). See link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/cli_tools/openshift-cli-oc#installing-openshift-cli[Installing the OpenShift CLI^].
+* You are familiar with link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/monitoring/configuring-the-monitoring-stack#creating-user-defined-workload-monitoring-configmap_configuring-the-monitoring-stack[creating a config map] for monitoring a user-defined workflow. You will perform similar steps in this procedure.
+* You have set the TrustyAI component in the DSC of your OpenDataHub or RHOAI instance set to `Managed`.
+
+.Procedure
+
+. Define a `ConfigMap` object in a YAML file called  to specify the `regexDetectorImage`. For example, create a YAML file called `regex_image_cm.yaml` with the following content:
++
+.Example `regex_image_cm.yaml`
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: gorch-sample-config
+data:
+  regexDetectorImage: 'https://quay.io/repository/trustyai/regex-detector/manifest/sha256:efab6cd8b637b9c35d311aaf639dfedee7d28de3ee07b412ab473deadecd3606' <1>
```

Comment:
@m-misiura just want to check, is this the correct quay image? Rui shared this on a working doc we have for 2.19 features: https://docs.google.com/document/d/1VP58VOoJVGxpwTis1nB9fFhk2GBfjk0zXwv2gB7wStc/edit?tab=t.px0e08jhdrd#heading=h.snusvohzjsx9

Reply:
Got it. I am also using this vllm image, formatting adjusted per your comment here :)